python/ray/autoscaler/updater.py

```diff
@@ -244,8 +244,11 @@ def run(self,
             port_forward=None,
             with_output=False,
             ssh_options=None):
-        if not ssh_options:
-            ssh_options = self.default_ssh_options
+        ssh_options = ssh_options or self.base_ssh_options
+
+        assert isinstance(ssh_options,
+                          SSHOptions), "ssh_options must be of type SSHOptions"
```

Comment:
nit: (no need to fix this time) - "ssh_options must be of type SSHOptions, got {type(ssh_options)}"

Reply:
Okay! Will do in the future!